CHANGELOG.md

```diff
@@ -31,7 +31,8 @@ All versions prior to 0.0.9 are untracked.
   ([#507](https://github.com/pypa/pip-audit/pull/507))
 
 * `pip-audit`'s handling of dependency resolution has been significantly
-  refactored and simplified ([#523](https://github.com/pypa/pip-audit/pull/523))
+  refactored and simplified ([#523](https://github.com/pypa/pip-audit/pull/523)
```

Comment:
Let's break this into a separate CHANGELOG entry, now that the previous entry is part of a past release.

Reply:
[05180d3](https://github.com/pypa/pip-audit/pull/540/commits/05180d3f9f36e643958571bd076104f3a252a798)